### ISSUE TITLE
[Suggestion] Extend the time frame to perform the double jump?

### ISSUE THREAD

**Issue by TheLongLife** [beat-feet/beat-feet#116]
Nice little game.  
I played it for a bit, it's fun and dificult.  

I just think that having a longer time frame to perform a second jump would make it better?  
Considering the number of obstacles and the speed at which you're going, I think that having more control over your jumps would be sensible.

**Comment by tysino** [comment]
I agree. It just feels buggy that sometimes touches do nothing (*after* the double jump time frame but *before* you land on the ground). Some games solve this by always allowing a second jump (without timeout) and even highlighting it (e.g. by leaving a trail after the character or making it spin, etc.), so the user knows he has no air jump left.

**Comment by pserwylo** [comment]
Thanks for the feedback @TheLongLife and @tysino.

I've created a [v0.13.0 and tagged it](https://github.com/beat-feet/beat-feet/releases/tag/v0.13.0), ready for F-Droid to build and release over the coming days (usually no longer than a week).

Please let us know when it comes out if you think that it is better/worse/etc. Any feedback or constructive criticism more than welcome.

Here is the release notes:

> 🦘 More generous double jumps.
> 
> * Allow the second jump a greater time after the first jump.
> * Show a different image for the second jump to remind you that you've used the second jump.
> 
> Thanks to those who provided feedback regarding this on GitHub.

**Comment by tysino** [comment]
Wow that was fast as hell, what a *commit*ment. F-Droid just compiled the newest version and it's definitely better. The ducking image is a nice touch.

Though i would still prefer no timeout for the double jump it is at some point just a matter of taste and a respectable design desicion. Especially since you argumented "Only allow double jumps when you are close to the top of your first jump. Seems to offer a nice experience."

Congrats on your code base btw, seems like you found a clean structure to do changes in just a few lines.

**Comment by pserwylo** [comment]
Haha, glad you were not horrified at the code base. Thanks for providing the feedback on this feature too. 

FYI keep an eye out, I'm now working on adding extra "Worlds" (really just new collections of songs), and then will work on letting community members host their own levels (JSON metadata and MP3 files) online. 

This latest change got me back interested in this again after a break to work on other games and apps...
